MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/javaservices/MultipleGeometriesDirectionsRouteActivity.java

```diff
@@ -0,0 +1,289 @@
+      .build();
+
+    directionsApiClient.enqueueCall(new Callback<DirectionsResponse>() {
+      @Override
+      public void onResponse(Call<DirectionsResponse> call, Response<DirectionsResponse> response) {
+        System.out.println(call.request().url().toString());
+
+        // You can get the generic HTTP info about the response.
+        Timber.d("Response code: " + response.code());
+        if (response.body() == null) {
+          Timber.e("No routes found, make sure you set the right user and access token.");
+          return;
+        } else if (response.body().routes().size() < 1) {
+          Timber.e("No routes found");
+          return;
+        }
+
+        if (response.body() != null) {
+
+          // Get the directions route
+          currentRoute = response.body().routes().get(0);
+
+          mapboxMap.getStyle(new Style.OnStyleLoaded() {
+            @Override
+            public void onStyleLoaded(@NonNull Style style) {
+              // Retrieve the sources from the map
+              GeoJsonSource circleLayerSource = style.getSourceAs(CIRCLE_GEOJSON_SOURCE_ID);
+              GeoJsonSource lineLayerSource = style.getSourceAs(LINE_GEOJSON_SOURCE_ID);
+
+
+              if (circleLayerSource != null && response.body() != null) {
+
+                List<Feature> featureList = new ArrayList<>();
+
+                // Use each step maneuver's location to create a Point Feature.
+                // The Feature is then added to the list.
+                if (currentRoute.legs().size() > 0) {
+                  for (LegStep singleRouteLeg : currentRoute.legs().get(0).steps()) {
+                    Point stepManeuverLocationPoint = singleRouteLeg.maneuver().location();
+                    featureList.add(Feature.fromGeometry(stepManeuverLocationPoint));
+                  }
+                } else {
+                  Timber.d("%s", getString(R.string.no_legs_toast));
+                }
+
+                // Update the CircleLayer's source with the Feature list.
+                circleLayerSource.setGeoJson(FeatureCollection.fromFeatures(featureList));
+
+                // Update the LineLayer's source with the Polyline route from the Directions API response.
+                if (lineLayerSource != null && currentRoute.geometry() != null) {
+                  lineLayerSource.setGeoJson(Feature.fromGeometry(LineString.fromPolyline(
+                    currentRoute.geometry(), PRECISION_6)));
+                }
+
+                // Ease the camera to fit to the Directions route.
+                easeCameraToShowEntireDirectionsRoute(new LatLng(origin.latitude(), origin.longitude()),
+                  new LatLng(destination.latitude(), destination.longitude()));
+              }
+            }
+          });
+        }
+      }
+
+      @Override
+      public void onFailure(Call<DirectionsResponse> call, Throwable throwable) {
+        Timber.e("Error: " + throwable.getMessage());
```

Comment:
either use `Timber.e("Error: %s", throwable.getMessage())` or use the throwable directly (Timber exposes an overload for that)

Reply:
Fixed to `Timber.e(throwable);`